src/ui/Omnibox/Omnibox.ts

```diff
@@ -271,7 +272,13 @@ export class Omnibox extends Component {
      *
      * **Default:** `false`
      */
-    clearFiltersOnNewQuery: ComponentOptions.buildBooleanOption({ defaultValue: false })
+    clearFiltersOnNewQuery: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+    /**
+     * The amount of focus time (in milliseconds) required on a query suggestion before requesting a preview of its top results.
+     *
+     * **Default:** `200`
+     */
+    executePreviewsQueryDelay: ComponentOptions.buildNumberOption({ defaultValue: 200 })
```

Comment:
As discussed live, this option does not make sense on the omnibox.

This is controlling the behaviour of component A through configuration of component B. This is not what we want.

QuerySuggestPreview options should be self contained.

Reply:
Good point, I'm changing it to an event to avoid circular dependencies. The alternative would be to have `QuerySuggestPreview` manage the timeout while fetching previews and change the `IPopulateSearchResultPreviewsEventArgs` event arguments to only take a single promise instead, but that sounded a little riskier. If you prefer this solution, I can change it.